config.ts

```diff
@@ -18,6 +18,7 @@ interface PoolI {
     label: string
     link?: string
   }[]
+  version: 2 | 3
```

Comment:
Can we add the version to the validation with yup as well?

Reply:
Good idea 👍

Reply:
I'll add this in the integration PR, since it also requires the pool-config update.